src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -4171,6 +4176,20 @@ def _get_linux_os_config(file_path):
     return config_object
 
 
+def _get_http_proxy_config(file_path):
+    hp_config = get_file_json(file_path)
```

Comment:
better to do some validation on whether the file_path exist.

Reply:
good point, mde the changes